### ISSUE TITLE
RequestAuthorization method does not exist

### ISSUE THREAD

**Issue by suejon** [beaconinside/cordova-sdk#3]
From the Callbacks with meta data example, when trying to perform an Android build, the window.requestAuthorization() method cannot be found.

Here is the error log produced:

> app.js:20 Uncaught TypeError: window.requestAuthorization is not a function
    at app.js:20
    at Array.<anonymous> (ionic.bundle.js:56238)
    at Channel.onPlatformReady (ionic.bundle.js:2496)
    at Channel.fire (cordova.js:846)
    at cordova.js:231

**Comment by rabschi** [comment]
True, the requestAuthorization method is only working on iOS platform at the moment. The solution is to ignore it currently and ask on your own for location and notifications permissions which is also the recommended way. This way you keep in control how you ask for the user's permission. 

We'll fix the Android version so no errors appear.
